lib/db/schema/patch-031-032.sql

```diff
@@ -0,0 +1,94 @@
+CREATE TABLE IF NOT EXISTS securityEventNames (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  UNIQUE INDEX securityEventNamesUnique(name)
+) ENGINE=InnoDB;
+
+CREATE TABLE IF NOT EXISTS securityEvents (
+  uid BINARY(16) NOT NULL,
+  nameId INT NOT NULL,
+  FOREIGN KEY (nameId) REFERENCES securityEventNames(id) ON DELETE CASCADE,
+  tokenId BINARY(32),
+  verified BOOLEAN,
+  ipAddr BINARY(32) NOT NULL,
+  createdAt BIGINT SIGNED NOT NULL
+) ENGINE=InnoDB;
+
+INSERT INTO
+    securityEventNames (name)
+VALUES
+    ("account.create"),
+    ("account.login"),
+    ("account.confirm"),
```

Comment:
Do we need a "confirm" event if we're separately noting that the session was verified?